src/js/view/view.js

```diff
@@ -736,6 +736,12 @@ define([
                 return;
             }
             var transformScale = provider.resize(width, height, _model.stretching);
+            if(_model.stretching && !utils.hasClass(_playerElement, 'jw-stretch-' + _model.stretching)) {
+                utils.removeClass(_playerElement,
+                    'jw-stretch-none jw-stretch-uniform jw-stretch-fill jw-stretch-exactfit');
```

Comment:
Since this is meant to directly mirror the state of the model, we can break it out of the resize function.

In the View constructor
```
    // set initial state
    this.onStretchChange(_model, _model.get('stretching');
    // watch for changes
    _model.on('change:stretching', this.onStretchChange, this);
```

Then when you define "onStretchChange", you can use oldVal and newVal to modify the classes, instead of a whole list of possibilities